doc/source/conf.py

```diff
@@ -12,7 +12,7 @@
 DOC_SOURCE_DIR = pathlib.Path(__file__).parent.parent
 DOC_DIR = DOC_SOURCE_DIR.parent
 BASE_DIR = DOC_SOURCE_DIR.parent
-ACTIONS_PREFIXES = ("build-", "check-", "doc-", "release-", "tests-")
+ACTIONS_PREFIXES = ("build-", "check-", "doc-", "release-", "tests-", "hk-")
```

Comment:
Just my OCD trying to order prefixes in alphabetic order:

```suggestion
ACTIONS_PREFIXES = ("build-", "check-", "doc-", "hk-", "release-", "tests-")
```